src/lib/containers/AccessApprovalsTable.tsx

```diff
@@ -0,0 +1,142 @@
+import moment from 'moment'
+import React, { useState } from 'react'
+import { Button, Table } from 'react-bootstrap'
+import { SMALL_USER_CARD } from '../utils/SynapseConstants'
+import {
+  AccessApprovalSearchRequest,
+  AccessApprovalSearchSort,
+  AccessApprovalSortField,
+  Direction,
+  SortDirection,
+} from '../utils/synapseTypes'
+import Typography from '../utils/typography/Typography'
+import UserCard from './UserCard'
+import { PRODUCTION_ENDPOINT_CONFIG } from '../utils/functions/getEndpoint'
+import { useSearchAccessApprovalsInfinite } from '../utils/hooks/SynapseAPI/dataaccess/useAccessApprovals'
+import { SynapseSpinner } from './LoadingScreen'
+import SortIcon from '../assets/icons/Sort'
+import { formatDate } from '../utils/functions/DateFormatter'
+
+export type AccessApprovalsTableProps = {
+  accessorId: string
+  accessRequirementId?: string
+}
+export const AccessApprovalsTable: React.FunctionComponent<
+  AccessApprovalsTableProps
+> = ({ accessorId, accessRequirementId }) => {
+  const [isDescending, setIsDescending] = useState(false)
+  const [sort, setSort] = useState<AccessApprovalSearchSort[]>()
+
+  const searchRequest: AccessApprovalSearchRequest = {
+    accessorId,
+    accessRequirementId,
+    sort,
+  }
+
+  const { data, hasNextPage, fetchNextPage, isFetching } =
+    useSearchAccessApprovalsInfinite(searchRequest)
+  const accessApprovals = data?.pages.flatMap(page => page.results) ?? []
+  const onSort = (field: AccessApprovalSortField, isDescending: boolean) => {
+    const direction: SortDirection = isDescending
+      ? ('DESC' as SortDirection)
+      : ('ASC' as SortDirection)
+    setSort([{ field, direction }])
+    setIsDescending(!isDescending)
+  }
+  return (
+    <div className="AccessApprovalsTable bootstrap-4-backport">
+      <Typography style={{ padding: '12px' }} variant="headline3">
+        Status in Access Requirements
+      </Typography>
+      <Table striped borderless bordered={false}>
+        <thead className="access-approval-header">
+          <tr>
+            <th className="thead-cell remove-border">AR ID</th>
+            <th className="thead-cell remove-border">
+              Access Requirement Name
+            </th>
+            <th className="thead-cell remove-border">Submitter</th>
+            <th className="thead-cell remove-border">Status</th>
+            <th className="thead-cell remove-border">
+              Modified Date
+              <SortIcon
+                role="button"
+                style={{ float: 'right' }}
+                active={false}
+                direction={
+                  sort && sort[0].field === 'MODIFIED_ON'
+                    ? sort[0].direction === 'DESC'
```

Comment:
No longer undefined or an array
```suggestion
                  sort.field === 'MODIFIED_ON'
                    ? sort.direction === 'DESC'
```